### ISSUE TITLE
PolyPen: Unhandled Exception Caught: Exception caught

### ISSUE THREAD

**Issue by indiegael** [CGCookie/retopoflow#843]
Automatically Generated Info
-----------------------------------

```
Environment:

- RetopoFlow: 3.0.0β2
- Blender: 2.83.0 master 2020-06-03
- Platform: Windows, 10, 10.0.18362, AMD64, AMD64 Family 21 Model 2 Stepping 0, AuthenticAMD
- GPU: ATI Technologies Inc., AMD Radeon R9 200 Series, 4.5.13596 Core Profile Context 20.5.1 27.20.1017.1011, 4.60
- Timestamp: 2020-06-10 14:04:43.079989
- Undo: insert, insert, select, tweak move, tweak move, tweak move, tweak move, tweak move, tweak move, tweak move

Error Hash: 0e6117104496d9f7fa8e351fa20df725

Trace:

- EXCEPTION (<class 'ValueError'>): faces.new(verts): face already exists
-     C:\Users\Gael\AppData\Roaming\Blender Foundation\Blender\2.83\scripts\addons\RetopoFlow\retopoflow\rfmesh\rfmesh.py
- 000 1498:new_face() bmf = self.bme.faces.new(verts)
- 001 1600:clean_duplicate_bmedges() mapping[bmf] = self.new_face(lbmv)
-     C:\Users\Gael\AppData\Roaming\Blender Foundation\Blender\2.83\scripts\addons\RetopoFlow\retopoflow\rf\rf_target.py
- 002 0483:clean_duplicate_bmedges() return self.rftarget.clean_duplicate_bmedges(vert)
-     C:\Users\Gael\AppData\Roaming\Blender Foundation\Blender\2.83\scripts\addons\RetopoFlow\retopoflow\rftool_polypen\polypen.py
- 003 0409:_insert() self.rfcontext.clean_duplicate_bmedges(bmv1)
-     C:\Users\Gael\AppData\Roaming\Blender Foundation\Blender\2.83\scripts\addons\RetopoFlow\retopoflow\rftool.py
- 004 0141:wrapper() ret = fn(*args, **kwargs)
-     C:\Users\Gael\AppData\Roaming\Blender Foundation\Blender\2.83\scripts\addons\RetopoFlow\retopoflow\rftool_polypen\polypen.py
- 005 0236:insert() return self._insert()
-     .../common\fsm.py
- 006 0054:run() return fn(*args, **kwargs)
```

**Comment by vxlcoder** [comment]
sorry it took so long to get back to this!

i think this issue _should_ be fixed with commit 16a4233.  i haven't been able to reproduce it, so it's difficult to test.

By chance, @indiegael, do you happen to remember what steps you took that resulted in this error?

**Comment by vxlcoder** [comment]
i'm closing this issue for now.

if you run into it again, please reopen and post reproduction steps.

Thanks!
